project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module

```diff
@@ -1098,5 +1098,5 @@ function c4m_user_og_invite_access($group = NULL) {
     return FALSE;
   }
 
-  return og_is_member($group->type, $group->nid, 'user', $user);
+  return og_is_member('node', $group->nid, 'user', $user);
```

Comment:
`$group->type` returns "group" instead of "node" and this causes `og_is_member()` to return false even if the user is indeed a member.